velox/docs/configs.rst

```diff
@@ -109,6 +109,11 @@ Expression Evaluation Configuration
      - bool
      - false
      - This flags forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
+   * - cast_string_to_date_non_standard
```

Comment:
Let's also document configs that alter cast behavior in cast documentation: https://facebookincubator.github.io/velox/functions/presto/conversion.html

Reply:
@mbasmanova "non_standard" here means non ISO8601 standard. Or change it to `cast_string_to_date_iso8601` and make the default value true?